presto-product-tests/README.md

```diff
@@ -35,78 +35,17 @@ broken.
     pip install docker-compose
     ```
 
-### OS X using Docker for Mac (macOS 10.10.3 Yosemite or newer) [PREFERRED WAY]
+### OS X using Docker for Mac
 
-* Install Docker for Mac: https://docs.docker.com/docker-for-mac/
+* Install [Docker for Mac](https://docs.docker.com/docker-for-mac/)
 
 * Add entries in `/etc/hosts` for all services running in docker containers:
 `hadoop-master`, `mysql`, `postgres`, `cassandra`, `presto-master`.
-They should point to your external IP address (shown by `ifconfig` on your Mac (not inside docker)).
+They should point to your external IP address (shown by `ifconfig` on your Mac, not inside Docker).
 
 * The default memory setting of 2GB might not be sufficient for some profiles like `singlenode-ldap`.
 You may need 4-8 GB or even more to run certain tests. You can increase Docker memory by going to
-Docker Preferences -> Advanced -> Memory.
-
-### OS X using Docker Toolbox (macOS 10.8 "Mountain Lion" or newer) [NOT RECOMMENDED]
```

Comment:
I wonder if there is some infra related code that we can now remove.

Reply:
I didn't see any, which makes sense to me since this seems to be an implementation detail of how `docker` and related commands work, but others would know better than me.